package.json

```diff
@@ -11,7 +11,6 @@
     },
     "dependencies": {
         "@mdi/font": "^7.3.67",
-        "@slipmatio/control-knob": "^0.1.0",
```

Comment:
Why are we removing the control knob are we planning on writing our own component for rotary knob

Reply:
It was causing errors with the components not being found. I think this is an issue with the package we were using. 
Also I'd really like to clean up the codebase and remove code that is not being used 👌 2 birds, 1 stone. 
https://github.com/DCC-EX/EX-WebThrottle2/pull/25/commits/29fa423317522ddeab9fa3616b3610ea6ebc2858

Reply:
p.s. see failing Playwright hobs on the `main` branch

Reply:
I knew they were failing i was still in the process of writing the component